core/templates/services/UpgradedServices.ts

```diff
@@ -1222,7 +1222,6 @@ export class UpgradedServices {
     upgradedServices['TranslationsBackendApiService'] =
       new TranslationsBackendApiService(
         upgradedServices['HttpClient']);
-
```

Comment:
Can you add back the new line? You removed it here.

Reply:
done!